src/main/flatbuffers/rlbot.fbs

```diff
@@ -41,13 +41,16 @@ struct Vector3 {
   z:float;
 }
 
-// Values are in radians
+/// Expresses the rotation state of an object in Euler angles, with values in radians.
 struct Rotator {
   pitch:float;
   yaw:float;
   roll:float;
 }
 
+/// Expresses the rotation state of an object.
+/// x, y, and z can be considered an axis of rotation, and w the amount of rotation.
```

Comment:
This comment isn't quite accurate. Example with w=0:

![](https://i.imgur.com/bfWdBPq.png)